src/Event/Http/HttpRequestEvent.php

```diff
@@ -94,13 +103,17 @@ public function getServerName(): string
 
     public function getPath(): string
     {
+        if (isset($this->event['rawPath'])) {
+            return $this->event['rawPath'];
+        }
```

Comment:
`rawPath` seems to exist in v2 only.

`path` exists in v1 only.

I think we should make that clearer, even if it's not as optimum as possible, because it will make maintenance easier. Something like this:

```
if ($this->payloadVersion >= 2) {
    return $this->event['rawPath'] ?? '/';
} else {
    return $this->event['path'] ?? '/';
}
```